internal/rpc/rpcserver/server.go

```diff
@@ -2041,12 +2041,14 @@ func (s *walletServer) ValidateAddress(ctx context.Context, req *pb.ValidateAddr
 		_, script := ka.PaymentScript()
 		result.PayToAddrScript = script
 
+		_, redeem := ka.RedeemScript()
```

Comment:
There will be updates for `stdscript` below why my PR goes in, but even with this version of txscript, it would be best to also return the script version and feed that to `ExtractPkScriptAddrs` below.

Reply:
Now following a pattern similar to the grpc method.